docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss

```diff
@@ -29,6 +29,8 @@
 .ptype-hs-accordion,
 .ptype-hs-postcard,
 .ptype-hs-hero-image,
+.ptype-hs-spotlight,
+.ptype-hs-banner,
```

Comment:
something is happening where if they are in a well, they get a margin top and bottom and the spacing looks too large: 
<img width="588" alt="Screen Shot 2020-08-07 at 12 40 06 PM" src="https://user-images.githubusercontent.com/12848168/89668020-27629200-d8ab-11ea-8b04-935d0c6ccb3c.png">

Reply:
we've decided that this regression should be solved elswhere